File: Algorithm/java/Searching/FindMin.java
package Algorithm.java.Searching;

public class FindMin {
    public static void main(String[] args){
        int[] arr = {7,1,22,56,-9,80};
        System.out.println(min(arr));
    }

    static int min(int[] arr){
        int ans = arr[0];
        for(int i =1; i < arr.length; i++){
            if(arr[i] < ans){
                ans = arr[i];
            }
        }
        return ans;
    }
    
}

File: Algorithm/Javascript/Searching/FindMin.js
const numbers = [56,9,12,34, 67, 7,12];

function findMin(arr){
    if(arr.length === 0){
        return undefined;
    }
    let min = arr[0];

    for(let i = 0; i < arr.length; i++){
        if(arr[i] < min){
            min = arr[i]
        }
    }
    return min;
}

console.log(findMin(numbers));
